core/domain/summary_services.py

```diff
@@ -743,3 +743,28 @@ def get_recently_published_exp_summary_dicts(limit):
         reverse=True)
 
     return get_displayable_exp_summary_dicts(summaries)
+
+
+def get_displayable_story_summary_dicts(story_summary):
+    """Returns a list of displayable story summary dicts for topic
+    viewer page.
+
+    Args:
+        story_summary: list(dict). A list of story summaries.
```

Comment:
The typeinfo is wrong. Proofread your docstrings please.

Reply:
Done.